### PR TITLE
CLDR-7536 Revise cldrNotify

### PR DESCRIPTION
**Pull request by btangmu**
-New method cldrNotify.exception for caught exceptions

-Make duration automatic: no time-out for errors/exceptions, 8 seconds for show/warning

-Minor refactoring and comments

CLDR-7536

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by btangmu** [comment]
enough preliminary stuff! after this I'll add more vue components to the info panel...

File: tools/cldr-apps/js/src/esm/cldrProgress.mjs [hunk 1]
```diff
@@ -122,11 +122,7 @@ function insertWidget(spanId) {
     console.error(
       "Error loading ProgressMeters vue " + e.message + " / " + e.name
     );
-    cldrNotify.error(
-      `${e.name} while loading ProgressMeters.vue`,
-      e.message,
-      cldrNotify.NO_TIMEOUT
-    );
+    cldrNotify.exception(e, "while loading ProgressMeters.vue");
```
Comment (srl295) [inline tools/cldr-apps/js/src/esm/cldrProgress.mjs:125]: Same. I wonder if we can refactor some functions around the mount() calls.   But in any event, this is great progress.

**Review by srl295 — APPROVED** [review]

File: tools/cldr-apps/js/src/esm/cldrProgress.mjs (hunk 1 above)
Comment (btangmu) [reply tools/cldr-apps/js/src/esm/cldrProgress.mjs:125]: Yes, I'll do some refactoring around mount() and also prefix the exception() message with "Internal error"... who knows what e.name would even be if mount() ever throws an exception; I'll try to force that to happen in testing, to see what it looks like

Comment (btangmu) [reply tools/cldr-apps/js/src/esm/cldrProgress.mjs:125]: @srl295 Here's how cldr.exception looks with added "Internal error":

image

OK?

**Pull request merged**